src/Events.js

```diff
@@ -43,19 +45,34 @@
         defaultAppName = defaults && defaults.appName || "*";
         cloneData = (defaults && typeof defaults.cloneEventData === "boolean" ? defaults.cloneEventData : false);
         eventBufferLimit = (defaults && !isNaN(defaults.eventBufferLimit) ? defaults.eventBufferLimit : -1);
+        rethrow = (defaults &&  defaults.rethrow === false ? false : true);
+        async = (defaults && defaults.async === true ? defaults.async : false);
 
         /**
          * This registers to an event only once, if it has fired the bind will be removed
-         * @param data
+         * @param app = {
+         *   eventName: string that is the name of the event that will be triggered like 'click'
+         *   aSync: boolean flag if this call back is called synchronously when the event fires, or after we queue all the listeners
+         *   appName: string that specifies an added identifier for multiple instances of the same event name (click by button1, click by button 2)
+         *   func: function - the callback function which the event data will be passed to
+         *   context: the context which the event data will be run with
+         *   triggerOnce: this is for listening only to the first trigger of this event
+         *   } || app = app name
+         *
+         * @param ev = event name
+         * @param fn = callback function
          * @return {*}
          */
-        function once(data) {
-            if (data) {
-                data.triggerOnce = true;
-                return bind(data);
-            } else {
-                return null;
+        function once(app, ev, fn) {
```

Comment:
not having the multiple params option would be a big mistake in my opinion.
as a user of the API i dont want to create an object every time i want to call bind or once. especially with the default app name when typically i just need to pass two parameters, i doesnt make sense to make me create an object literal every time.